src/tutorials/shift.md

```diff
@@ -107,3 +107,9 @@ To use a custom prompt, click on the `+` button within the message input field,
 Shift is also able to automatically rename Replay session tabs from their numerical names to descriptive names that identify the purpose of the request. To enable this feature, and configure additional settings, navigate to the `AI Session Renaming` tab.
 
 <img alt="Session tab renaming settings." src="/_images/shift_session_renaming.png" center />
+
```

Comment:
There are two shift-related chapters in the video, one for shift core features, one for agents.

Can we each under the corresponding sections of this page?